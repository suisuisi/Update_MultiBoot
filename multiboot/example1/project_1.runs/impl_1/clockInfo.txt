-------------------------------------
| Tool Version : Vivado v.2024.1
| Date         : Thu Sep 12 09:20:46 2024
| Host         : MININT-7LTFL4B
| Design       : design_1
| Device       : xc7k325t-ffg900-2--
-------------------------------------

For more information on clockInfo.txt clock routing debug file see https://support.xilinx.com/s/article/000035660?language=en_US

