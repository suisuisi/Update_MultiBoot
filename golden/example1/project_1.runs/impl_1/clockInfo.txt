-------------------------------------
| Tool Version : Vivado v.2024.1
| Date         : Thu Sep 12 09:17:03 2024
| Host         : MININT-7LTFL4B
| Design       : design_1
| Device       : xc7k325t-ffv900-2--
-------------------------------------

For more information on clockInfo.txt clock routing debug file see https://support.xilinx.com/s/article/000035660?language=en_US

